README.md

```diff
@@ -27,15 +27,15 @@ CONSOLE_URL=https://<machine-ip>:8443 \
 ```bash
 $ ./docker_build.sh
 # or
-$ docker build -t protractor-smoke-test .
+$ docker build -t origin-web-console-smoke-test .
 ```
 
 # Run the tests in a container
 
 ```bash
```

Comment:
Shall we add `TEST_INTERVAL` to README?  Assuming its in `minutes`, unless we wanted to support something like `TEST_INTERVAL=2m` or `TEST_INTERVAL=1h`, but I dont think thats likely necessary.